ProcessMaker/Http/Controllers/Api/TaskController.php

```diff
@@ -330,4 +331,19 @@ public function getScreenFields(ProcessRequestToken $task)
             return response()->json(['error' => 'Screen not found'], 404);
         }
     }
+
+    /**
+     * Returns a comma-separated list of user IDs that are members of the groups specified in the "groups" query parameter.
+     *
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function getAssignedUsersInGroups(Request $request)
+    {
+        $groups = $request->input('groups');
+        $users = GroupMember::whereIn('group_id', $groups)->pluck('member_id');
```

Comment:
Could you add another where clause to specify the member_type? Since from what I see, it could be either a User::class or a Group::class. (Maybe this isn’t necessary. Please check my comment below regarding nested groups).

Reply:
My `Requesters` group has one user. This function correctly retrieves that user from the group, but my `Requesters` group also has another group assigned. I think it should be able to retrieve both the users from the Requesters group and the users from the nested group as well?

Reply:
Thank you, this has been resolved!